src/python/pants/notes/2.0.x.rst

```diff
@@ -5,6 +5,108 @@ This document describes releases leading up to the ``2.0.x`` ``stable`` series.
 
 See https://www.pantsbuild.org/v2.0/docs/release-notes-2-0 for an overview of the changes in this release.
 
+2.0.0b1 (9/23/2020)
+-------------------
+
+API Changes
+~~~~~~~~~~~
+
+* Default to setting `LC_ALL` and `LANG` in subprocesses again (#10800)
+  `PR #10800 <https://github.com/pantsbuild/pants/pull/10800>`_
+
+* Default to `[python-infer].inits = false` (#10813)
+  `PR #10813 <https://github.com/pantsbuild/pants/pull/10813>`_
+
+* Deprecate `--interpreter-constraints` option where it no-ops (#10833)
+  `PR #10833 <https://github.com/pantsbuild/pants/pull/10833>`_
+
+* Upgrade setuptools from 49.6.0 to 50.3.0 and wheel from 0.31.1 to 0.35.1 (#10803)
+  `PR #10803 <https://github.com/pantsbuild/pants/pull/10803>`_
+
+New Features
+~~~~~~~~~~~~
+
+* Restore support for third-party requirements with MyPy (#10811)
+  `PR #10811 <https://github.com/pantsbuild/pants/pull/10811>`_
+
+Bugfixes
+~~~~~~~~
+
+* Plumb the custom certs file through to Pip (via Pex) (#10837)
+  `PR #10837 <https://github.com/pantsbuild/pants/pull/10837>`_
+
+* Fix MyPy with Python 2-only third-party requirements (#10820)
+  `PR #10820 <https://github.com/pantsbuild/pants/pull/10820>`_
+
+* Fix `setup_py.with_binaries()` to automatically inject `dependencies` (#10829)
+  `PR #10829 <https://github.com/pantsbuild/pants/pull/10829>`_
+
+* Fix run not working with foo:main entry point. (#10821)
+  `PR #10821 <https://github.com/pantsbuild/pants/pull/10821>`_
+
+* Fix `internal_only` PEXes runtime interpreter diverging from buildtime (#10818)
+  `PR #10818 <https://github.com/pantsbuild/pants/pull/10818>`_
+
+* Local process cache validates that digests exist locally before hitting (#10789)
+  `PR #10789 <https://github.com/pantsbuild/pants/pull/10789>`_
+
+* Better Ctrl-C handling in the `test --debug` case (#10835)
+  `PR #10835 <https://github.com/pantsbuild/pants/pull/10835>`_
```

Comment:
Not sure if this is the right spot either. It's maybe an "improvement"? It wasn't necessarily broken before, just not ergonomic.